pkg/util/fargate/hostname_process.go

```diff
@@ -56,3 +58,17 @@ func getEKSHost(context.Context) (string, error) {
 	// Use the node name as hostname
 	return GetEKSFargateNodename()
 }
+
+func getECSManagedInstancesHost(ctx context.Context) (string, error) {
+	client, err := ecsmeta.V4FromCurrentTask()
+	if err != nil {
+		log.Debugf("error while initializing ECS metadata V4 client: %s", err)
+		return "", err
+	}
+
+	taskMeta, err := client.GetTask(ctx)
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("fargate_task:%s", taskMeta.TaskARN), nil
```

Comment:
I see `fargate_task` here but `managed-instance-task` below. I'm wondering if even though it feels wrong if  keeping it as`fargate_task` makes more sense instead of supporting a new prefix